docs/4.0/aws_oss_guide.md

```diff
@@ -125,12 +124,9 @@ more than happy to help you architect, setup and deploy Teleport into your envir
 
 We have these options for you. 
 
-- [Using our AWS Marketplace (Manual Setup)](##single-oss-teleport-amis-manual-gui-setup)
-- Using our AWS Marketplace (Cloudformation Setup)
-- Building your own base image (Manual)
-- Deploying with Cloudformation
-- Deploying with Terraform HA
-- Deploying with Terraform HA + Monitoring 
+- [Using our AWS Marketplace (Manual Setup)](#single-oss-teleport-amis-manual-gui-setup)
```

Comment:
Not sure the `our` works here - probably just `Using AWS Marketplace` would be better.